acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -126,25 +176,34 @@ private void pullPantheonImage() {
   }
 
   private String createPantheonContainer(final NodeConfiguration config) {
-    final HostConfig portBindingConfig =
-        HostConfig.newHostConfig().withPortBindings(tcpPortBinding(config), wsPortBinding(config));
+    @SuppressWarnings("unstable")
+    final String genesis = Resources.getResource(config.getGenesisFile()).getPath();
```

Comment:
may want to confirm this works under windows - some resource calls work, others don't - check the staticnodes test from pantheon (these were repaired to work under windows). (though equally, not sure when/how docker gets involved in all this)

Reply:
ah yes, windows 😞